src/components/glow/GlowWrapper.module.scss

```
@@ -0,0 +1,34 @@
+@use 'theme';
+
+$padding: var(--amino-glow-wrapper-padding);
+$border-radius: var(--amino-glow-wrapper-border-radius);
+$background: var(--amino-glow-wrapper-background);
+
+.wrapper {
+  z-index: 1;
+  position: relative;
+  overflow: hidden;
+  padding: $padding;
+  box-shadow: inset 0px 0px 1px 1px theme.$amino-gray-200;
+  border-radius: $border-radius;
+
+  ::after {
+    content: '';
+    z-index: -1;
+    position: absolute;
+    width: min(100%, 400px);
+    height: min(100%, 400px);
+    top: calc(var(--y, 0) * 1px - min(50%, calc(400px / 2)));
```

Since now we are refactoring this. I think it's better to always assign the css variable to a scss variable on the top, so we have only one place to control all of the available css variables without scrolling down to check for it.
Something like:
```suggestion
// Some where on top
$y: var(--y, 0);
$x: var(--x, 0);
...
    top: calc(var(y, 0) * 1px - min(50%, calc(400px / 2)));
```